TaffoInitializer/IndirectCallPatcher.cpp

```diff
@@ -0,0 +1,174 @@
+#include "IndirectCallPatcher.h"
+
+#include "Metadata.h"
+#include "TaffoInitializerPass.h"
+#include "TypeUtils.h"
+#include "llvm/ADT/Statistic.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Debug.h"
+#include <llvm/IR/IRBuilder.h>
+#include <llvm/Transforms/Utils/Cloning.h>
+#include <llvm/Transforms/Utils/ValueMapper.h>
+
+using namespace taffo;
+using namespace llvm;
+
+/// Check recursively whether an unsupported function is called.
+bool containsUnsupportedFunctions(llvm::Function *function, int exploreDepth) {
```

Comment:
being this function a `bool check...()` I would expect it to preserve the checked object.
Is there a reason why parameter `function` is not const?

Reply:
Agree with having it as a `const`